java-squid/src/main/java/org/sonar/java/se/ExplodedGraphWalker.java

```diff
@@ -100,6 +99,7 @@ public MaximumStepsReachedException(String s) {
 
   public ExplodedGraphWalker(JavaFileScannerContext context) throws MaximumStepsReachedException {
     alwaysTrueOrFalseChecker = new ConditionAlwaysTrueOrFalseCheck();
+    DEBUG_MODE_ACTIVATED = false && context.getFile().getName().contains("RuleNormalizer");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the literal "false" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1125)